inc/views/base/warnings/warnings.twig

```diff
@@ -0,0 +1,38 @@
+{% extends 'layouts/master.twig' %}
+
+{% block head %}
+    <title>{{ mybb.settings.bburl }} - {{ lang.warning_log }}</title>
+{% endblock head %}
+
+{% block body %}
+    {{ multipage|raw }}
+    <table border="0" cellspacing="{{ theme.borderwidth }}" cellpadding="{{ theme.tablespace }}" class="tborder">
+        <thead>
+            <tr>
+                <td class="thead" colspan="5">
+                    {% if warning_level > 0 %}
+                        <div class="float_right">{{ trans('current_warning_level', warning_level, user.warningpoints, mybb.settings.maxwarningpoints)|raw }}</div>
```

Comment:
I wonder if it's worth marking the `trans` function as HTML safe... There are quite a few translations that have HTML in them unfortunately.